Add DB_POSTGRESDB_IDLE_CONNECTION_TIMEOUT and DB_PING_INTERVAL_SECONDS db env variables (#3231)
n8n-io/n8n-docs: docs/hosting/configuration/environment-variables/database.md

| :------- | :---- | :------- | :---------- |
| `DB_TYPE`<br>/`_FILE` | Enum string:<br> `sqlite`, `postgresdb` | `sqlite` | The database to use. |
| `DB_TABLE_PREFIX` | * | - | Prefix to use for table names. |

> | `DB_PING_INTERVAL_SECONDS` | Number | `2` | The interval in seconds to ping the database to check if the connection is still alive. |

Reviewer: ```suggestion
| `DB_PING_INTERVAL_SECONDS` | Number | `2` | The interval, in seconds, between pings to the database to check if the connection is still alive. |
```

| `DB_POSTGRESDB_PASSWORD`<br>/`_FILE` | String | - | The PostgreSQL password. |
| `DB_POSTGRESDB_POOL_SIZE`<br>/`_FILE` | Number | `2` | Control how many parallel open Postgres connections n8n should have. Increasing it may help with resource utilization, but too many connections may degrade performance. |
| `DB_POSTGRESDB_CONNECTION_TIMEOUT`<br>/`_FILE` | Number | `20000` | Postgres connection timeout (ms).

> | `DB_POSTGRESDB_IDLE_CONNECTION_TIMEOUT`<br>/`_FILE` | Number | `30000` | Amount of time before an idle connection is eligible for eviction due to idle time.

Reviewer: ```suggestion
| `DB_POSTGRESDB_IDLE_CONNECTION_TIMEOUT`<br>/`_FILE` | Number | `30000` | Amount of time before an idle connection is eligible for eviction for being idle.
```
